src/objclass/objclass.h

```diff
@@ -153,7 +153,7 @@ extern int cls_get_snapset_seq(cls_method_context_t hctx, uint64_t *snap_seq);
 
 /* gather */
 extern int cls_cxx_gather(cls_method_context_t hctx, const std::set<std::string> &src_objs, const std::string& pool,
-			  const char *cls, const char *method, bufferlist& inbl);
+			  const char *cls, const char *method, bufferlist& inbl) __attribute__ ((deprecated));
```

Comment:
Hmm, I don't see the doc part (release notes) on deprecating the method. If it's already in, a mention in the commit msg should be present.

Reply:
@rzarzynski fixed it, it was not picked so i added it

Reply:
Oh, sorry. I know why git has skipped this line: it's already present – https://github.com/ceph/ceph/pull/57819/files#diff-e5807f35e3c85ea0c57bba6c6713ace9a5f379edd4ec033f0ac2ac81ccad9972R188. Let's go with a mention in the commit message.

Reply:
@rzarzynski are you approving it?